client/src/components/NavBar.js

```diff
@@ -24,32 +21,42 @@ const styles = {
   },
   actionItem: {
     color: 'white',
+    transition: 'all .3s ease-in-out',
+    '&:hover': {
+      textDecoration: 'none',
+      color: 'white',
+      transform: 'scale(1.1)',
+    },
     '&$selected': {
       color: 'white',
+      backgroundColor: 'red',
     },
   },
   selected: {},
   buttons: {
     fontSize: '40px',
+
+    transition: 'all .3s ease-in-out',
+    '&:hover': {
+      transform: 'scale(1.1)',
+    },
   },
 };
 
 class NavBar extends React.Component {
-  // state = {
-  //   value: 'log',
-  // };
+  state = {
+    value: 'log',
+  };
 
-  // handleChange = (event, value) => {
-  //   console.log(value);
-  //   this.setState({ value });
-  // };
+  handleChange = (event, value) => {
```

Comment:
What is the motivation to take the navigation out of redux again?

Reply:
If by accident, this should be reverted. If on purpose this should go into a new PR

Reply:
Was this resolved? The Files Changed I'm seeing still has this PR removing redux